src/services/dispatchers.ts

```diff
@@ -13,15 +13,19 @@ export async function sync(sdk: OAuth1Client, path: string) {
   const rql = rqlBuilder().eq('tags', cliManagedTag).build();
   const exhDispatchers = await dispatcherRepository.findAll(sdk, rql);
 
+  // Ensure all Dispatchers and actions have name fields
   for (const localDispatcher of localDispatchers) {
-    console.group(blue(`Synchronizing Dispatcher: ${localDispatcher.name}`));
-
-    // Ensure all Dispatchers and actions have name fields
     assertRequiredFields(localDispatcher);
+  }
+
+  for (const localDispatcher of localDispatchers) {
+    console.group(blue(`Synchronizing Dispatcher: ${localDispatcher.name}`));
 
     // TODO: This does not account for dispatchers that exist in Extra Horizon with an existing name, but not a EXH_CLI_MANAGED tag
     const exhDispatcher = exhDispatchers.find(({ name }) => name === localDispatcher.name);
     await synchronizeDispatcher(sdk, localDispatcher, exhDispatcher);
+
+    console.log(green(`Synchronized Dispatcher: ${localDispatcher.name} ✓`));
```

Comment:
It makes a bit more sense to have this in the same function as the synchronising log